site/docs/visual_elements/favicon.mdx

```diff
@@ -0,0 +1,161 @@
+![Dark Apple touch icon](../../static/favicon/thumb/dark_apple_touch_icon.png)| apple-touch-icon.png | .png          |
+|| android-chrome-256x256.png | .png          |
+|| mstile-150x150.png | .png          |
+|| safari-pinned-tab.svg | .svg          |
+
+### Transparent light favicons
+
+Favicon variant                                                                                                                               | Filename     | Format                        |
+--------------------------------------------------------------------------------------------------------------------------------------------- |---------------|-------------------------------|
+![Transp Light 16×16px favicon](../../static/favicon/thumb/light_transparent_favicon_16x16.png)| favicon-16x16.png | .png          |
+|| favicon-32x32.png | .png          |
+|| favicon.ico | .ico          |
+
+### Transparent light app icons
+
+App icon variant                                                                                                                               | Filename     | Format                        |
+--------------------------------------------------------------------------------------------------------------------------------------------- |---------------|-------------------------------|
+![Transp Light Apple touch icon](../../static/favicon/thumb/light_transparent_apple_touch_icon.png)| apple-touch-icon.png | .png          |
+|| android-chrome-192x192.png | .png          |
+|| mstile-150x150.png | .png          |
+|| android-chrome-256x256.png | .png          |
+
+### Transparent dark favicons
+
+Favicon variant                                                                                                                               | Filename     | Format                        |
+--------------------------------------------------------------------------------------------------------------------------------------------- |---------------|-------------------------------|
+![Transp Dark 16×16px favicon](../../static/favicon/thumb/dark_transparent_favicon_16x16.png)| favicon-16x16.png | .png          |
+|| favicon-32x32.png | .png          |
+|| favicon.ico | .ico          |
+
+### Transparent dark app icons
+
+App icon variant                                                                                                                               | Filename     | Format                        |
+--------------------------------------------------------------------------------------------------------------------------------------------- |---------------|-------------------------------|
+![Transp Dark Apple touch icon](../../static/favicon/thumb/dark_transparent_apple_touch_icon.png)| apple-touch-icon.png | .png          |
+|| android-chrome-192x192.png | .png          |
+|| mstile-150x150.png | .png          |
+|| android-chrome-256x256.png | .png          |
+
+
+## Creating Pinned Tab Icons
```

Comment:
After reading this chapter, I do not understand how this relates to favicons. Can the user use some of the favicons for this pinned site image? If yes, how? Maybe create a step by step guide of setting this up.

Reply:
The pinned image is included in the favicon package